src/token/components/TokenSelectDropdown.css

```diff
@@ -0,0 +1,4 @@
+.ock-scrollbar {
+  scrollbar-width: thin;
+  scrollbar-color: #d1d5db #ffffff;
+}
```

Comment:
scrollbar properties are not available on tailwind and we have to use a CSS class name here.

Reply:
We are going to merge this PR as it is. But I think for those special case, it might just be worth doing inline css. And just call it the day.

Like create a new CSS file just for a couple of line, I don't think it's worth it.